client/src/app/components/Header/MobileMenu/SideMenu.js

```diff
@@ -0,0 +1,72 @@
+import React from "react";
```

Comment:
This file looks the same as the other previous file. Did you forget to remove it?

Reply:
Yes, you're right. File has been deleted now